src/__tests__/acceptance/audit.mixin.acceptance.ts

```diff
@@ -0,0 +1,755 @@
+      where: {
+        id: {
+          inq: [mockItemArray[0].id, mockItemArray[1].id, mockItemArray[2].id],
+        },
+        itemName: 'replacedTestItem',
+        description: 'replacedTestDescription',
+      },
+    });
+    expect(storedRecords.length).to.equal(3);
+
+    //check if audit log is stored
+    const auditLog = await auditLogRepositoryInstance.find();
+    expect(auditLog.length).to.equal(0);
+  });
+
+  const auditLogErrorRepositoryInstance = new TestAuditLogErrorRepository();
+  const auditLogErrorRepository = sinon
+    .stub()
+    .resolves(auditLogErrorRepositoryInstance); //repository which will cause catch statements to execute
+  const testRepositoryAuditLogErrorInstance = new TestRepository(
+    testDataSourceInstance,
+    getCurrentUser,
+    auditLogErrorRepository,
+  );
+
+  it(`should log appropriate message when audit log can't be created on calling create method`, async () => {
+    const mockItem = getMockItem();
+    await testRepositoryAuditLogErrorInstance.create(mockItem);
+
+    expect(consoleMessage.includes(`Audit failed for data =>`)).to.be.true();
+    expect(consoleMessage.includes(`"actedAt":`)).to.be.true();
+    expect(consoleMessage.includes(`"actor":"testId"`)).to.be.true();
+    expect(consoleMessage.includes(`"action":"INSERT_ONE"`)).to.be.true();
+    expect(consoleMessage.includes(`"after":`)).to.be.true();
+    expect(consoleMessage.includes(`"id":`)).to.be.true();
+    expect(consoleMessage.includes(`"itemName":"testItem"`)).to.be.true();
+    expect(
+      consoleMessage.includes(`"description":"testDescription"`),
+    ).to.be.true();
+    expect(consoleMessage.includes(`"entityId":`)).to.be.true();
+    expect(consoleMessage.includes(`"actedOn":"TestModel"`)).to.be.true();
+    expect(consoleMessage.includes(`"actionKey":"Item_Logs"`)).to.be.true();
+  });
+  it(`should log appropriate message when audit log can't be created on calling createAll method`, async () => {
+    const mockItemArray = getMockItemArray();
+    await testRepositoryAuditLogErrorInstance.createAll(mockItemArray);
+
+    expect(consoleMessage.includes(`Audit failed for data =>`)).to.be.true();
+    expect(consoleMessage.includes(`"actedAt":`)).to.be.true();
+    expect(consoleMessage.includes(`"actor":"testId"`)).to.be.true();
+    expect(consoleMessage.includes(`"action":"INSERT_MANY"`)).to.be.true();
+    expect((consoleMessage.match(/"after":/g) ?? []).length === 3).to.be.true();
+    expect((consoleMessage.match(/"id":/g) ?? []).length === 3).to.be.true();
+    expect(
+      (consoleMessage.match(/"itemName":/g) ?? []).length === 3,
+    ).to.be.true();
+    expect(
+      (consoleMessage.match(/"description":/g) ?? []).length === 3,
+    ).to.be.true();
+    expect(consoleMessage.includes(`"entityId":`)).to.be.true();
+    expect(consoleMessage.includes(`"actedOn":"TestModel"`)).to.be.true();
+    expect(consoleMessage.includes(`"actionKey":"Item_Logs"`)).to.be.true();
```

Comment:
use expect deepEqual to and compare to an object instead of using these many expects.